docs/submitting-molecular-data.md

```diff
@@ -2,11 +2,86 @@
 id: submitting-molecular-data
 title: Submitting Molecular Data
 ---
+Molecular data will be submitted to your local Regional Data Processing Centre (RDPC). RDPCs are responsible for processing your programs molecular data according to the [Harmonization Pipeline](). If you are unsure what the correct RDPC is to submit to, please [contact the DCC](https://platform-ui.qa.argo.cancercollaboratory.org/contact).
 
-## Installing the Song and Score Clients
 
-## Formatting Molecular Metadata Payloads
+This guide will describe how to submit molecular data to the ARGO Data Platform. Molecular data consists of all raw data  files generated from your program's donors, as well as any associated file metadata.
 
-## Uploading Molecular Files
+## Data Submission Client Configuration
+Molecular data is uploaded to the ARGO Data Platform using the Song and Score CLIs (Command Line Clients).  Song is an open source system used to track and validate metadata about raw data submissions. Score securely and quickly manages upload and download of files to cloud repositories. The Song and Score clients are used in conjuction to  
```

Comment:
link song and score to the overture pages?

Reply:
Do you think it will confuse people to be directed to overture?

Reply:
hmmmmm yeah maybe. Ignore